src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,7 +577,23 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
-    
+
+    protected static List<FeedbackParticipantType> getParticipantListFromParams(String participantListParam) {
```

Comment:
@samsontmr I got that but only for one method do i need to create a new abstract class ?.Thats my question.